tests/formats/transition/README.md

```diff
@@ -0,0 +1,53 @@
+# Transition testing
+
+Transition tests to cover processing the chain across a fork boundary.
+
+## Encoding notes
+
+This test type contains objects that span fork boundaries. 
+In general, it may not be clear which objects belong to which fork so each 
+object is encoded with its corresponding fork version. 
+This `Version` should be used to select the appropriate version of the spec when interpreting the enclosed object. 
+
+```python
+class VersionedObject(Container):
+    fork_version: Version
```

Comment:
we can very easily just use a `Bytes32` and define the relationship in the `transition` test README

"this field is the sha256 of the utf-8 encoded string of the fork name"

Reply:
either bool or unique identifier works for me. Also in the transition test README, we need to add the fork sequential ordering

Reply:
I like the `VersionedObject` idea. It could probably be used in the fork choice rule and other tests as well.

I think except for Altair, other "fork names" are likely to be changed. (I haven't given up on renaming "phase0" to some more meaningful & everlasting name. 😄) In that context, using `fork_version` looks more ideal.

Reply:
yeah, im gonna look at just using a boolean to indicate pre/post fork in the data, then we can push which forks are which upstream and wont have to bother w/ data changing under our feet if that happens